### ISSUE TITLE
(request) Add Freeciv Web

### ISSUE THREAD

**Issue by gplastic** [Emupedia/emupedia.github.io#12]
Add Freeciv Web to EmuOS (Requires hosting a server afaik)

https://github.com/freeciv/freeciv-web

**Comment by ZaDarkSide** [comment]
Hm...is it emulating a specific Civilization game?

**Comment by gplastic** [comment]
Not necessarily, but it's based on and expands upon Civilization 2. It came out around the same time and it's a one of the most popular ways to play FOSS Civilization.

**Comment by ZaDarkSide** [comment]
So we need a dedicated server in order to run the game. For now we don't have a dedicated server to run this. I will leave the issue open for when we allocate one for this.

**Comment by gplastic** [comment]
ok

**Comment by WAGEOFFICIAL** [comment]
Can you add a banning system because people keep spamming on emu chat and it is getting annoying

**Comment by ZaDarkSide** [comment]
@WAGEOFFICIAL please open a new issue, this issue is for freeciv-web
